From 15ff0b0a9caeeea1d425e0bcf7370b98e1628eb2 Mon Sep 17 00:00:00 2001
From: Matthew Feickert <matthew.feickert@cern.ch>
Date: Thu, 13 Feb 2025 00:17:47 -0700
Subject: [PATCH] fix: Add '-ffixed-line-length-none' to FFLAGS

* Extend FFLAGS to be '-std=legacy -Wall -ffixed-line-length-none'
  to ensure that Fortran won't fail out given the very long path
  name conda-build gives to PREFIX.
---
 vendor/StdHEP/makefile               | 2 +-
 vendor/StdHEP/mcfio/src/GNUmakefile  | 2 +-
 vendor/StdHEP/src/make_opts          | 2 +-
 vendor/StdHEP/src/stdhep/GNUmakefile | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/vendor/StdHEP/makefile b/vendor/StdHEP/makefile
index c834e40..7e2496b 100644
--- a/vendor/StdHEP/makefile
+++ b/vendor/StdHEP/makefile
@@ -13,7 +13,7 @@ endif
 STDHEP_DIR = .
 BUILD_SHARED = false
 SHFLAG = -fPIC
-FFLAGS += -std=legacy
+FFLAGS += -std=legacy -Wall -ffixed-line-length-none
 CFLAGS += -Wno-implicit-function-declaration
 
 # The cern libraries are needed to build the Phase and Space executables.
diff --git a/vendor/StdHEP/mcfio/src/GNUmakefile b/vendor/StdHEP/mcfio/src/GNUmakefile
index 530ae9b..b7e8429 100644
--- a/vendor/StdHEP/mcfio/src/GNUmakefile
+++ b/vendor/StdHEP/mcfio/src/GNUmakefile
@@ -12,7 +12,7 @@ BINDIR = ../../bin
 
 SHFLAG = -fPIC
 
-FFLAGS += -std=legacy
+FFLAGS += -std=legacy -Wall -ffixed-line-length-none
 CFLAGS += -Wno-implicit-function-declaration -Wno-incompatible-pointer-types
 
 FINC = -I.
diff --git a/vendor/StdHEP/src/make_opts b/vendor/StdHEP/src/make_opts
index 85f4153..ea4bd92 100644
--- a/vendor/StdHEP/src/make_opts
+++ b/vendor/StdHEP/src/make_opts
@@ -8,5 +8,5 @@ MBITS := $(shell getconf LONG_BIT)
 # Earlier versions of Mac OS X need to add -lSystemStubs to DARWINLIBS
 DARWINLIBS=
 
-FFLAGS += -std=legacy
+FFLAGS += -std=legacy -Wall -ffixed-line-length-none
 CFLAGS += -Wno-implicit-function-declaration
\ No newline at end of file
diff --git a/vendor/StdHEP/src/stdhep/GNUmakefile b/vendor/StdHEP/src/stdhep/GNUmakefile
index af0a8d5..0231c90 100644
--- a/vendor/StdHEP/src/stdhep/GNUmakefile
+++ b/vendor/StdHEP/src/stdhep/GNUmakefile
@@ -7,7 +7,7 @@ STDHEP_DIR = ../..
 
 #this has been added by MZ
 FFLAGS+= -fd-lines-as-code -fPIE
-FFLAGS += -std=legacy
+FFLAGS += -std=legacy -Wall -ffixed-line-length-none
 CFLAGS += -Wno-implicit-function-declaration -Wno-incompatible-pointer-types
 
 SLIB = $(STDHEP_DIR)/lib
-- 
2.47.1

